frontend/src/components/form/field.jsx

```diff
@@ -0,0 +1,32 @@
+import { createFormControl } from "solid-forms";
+import { mergeProps, splitProps } from "solid-js";
+import ErrorMessage from "./error-message";
+
+const Field = (props) => {
```

Comment:
Please rename it to `Input`

Reply:
We renamed it to `Input` and reusing it at every input